### PR TITLE
Wyre

### PR DESCRIPTION
**Pull request by Marik-D**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

**Comment by JustCaptcha** [comment]
image

**Comment by JustCaptcha** [comment]
image

File: universal-login-react/src/config/index.ts [hunk 1]
```diff
@@ -13,6 +13,22 @@ const developmentConfig: Config = {
     baseUrl: 'https://api.ipdata.co',
     accessKey: 'c7fd60a156452310712a66ca266558553470f80bf84674ae7e34e9ee',
   },
+  network: 'ganache',
 };
 
-export default developmentConfig;
+const kovanConfig: Config = {
+  domains: ['mylogin.eth', 'poppularapp.eth', 'universal-id.eth'],
```
Comment (xitronix) [inline universal-login-react/src/config/index.ts]: we have unilogin.test on kovan now

File: universal-login-react/src/ui/Modals/ModalWrapper.tsx [hunk 2]
```diff
@@ -28,14 +28,16 @@ export const ModalWrapper = ({children, modalClassName, hideModal, message}: Mod
...
 
   return (
-    <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login universal-login-default'}>
-      <div className={classForComponent('modal-overlay')} onClick={hideModal} />
-      <div className={`${useClassFor('modal-wrapper')}`}>
-        <div className={classForComponent('modal')}>
-          {!!hideModal && <button onClick={hideModal} className={classForComponent('modal-close-btn')} />}
-          {children}
+    <div className="modal-backdrop">
+      <div className={modalClassName ? `universal-login ${modalClassName}` : 'universal-login universal-login-default'}>
```
Comment (xitronix) [inline universal-login-react/src/ui/Modals/ModalWrapper.tsx]: Does it necessary? @JustCaptcha

**Comment by xitronix** [comment]
image

I think iframe should cover all screen like ramp

**Comment by xitronix** [comment]
Should test it on mainnet

**Comment by Marik-D** [comment]
### Mainnet
Payment session does start fine, but I didn't complete the flow
### Apple pay
They do have a dropdown in their interface to select between Apple Pay and Debit Card.
Apple Pay should be working on Apple Devices

**Comment by xitronix** [comment]
We have delay between apeearing Wyre iframe and clicking pay: 
image
But i think we can add some window in jarvis (wyre initialization) in next pr

**Comment by Marik-D** [comment]
I think something got messed up during rebasing. There are files that don't belong here

**Comment by xitronix** [comment]
image
Notice message is missing

**Comment by JustCaptcha** [comment]
image

**Pull request merged**